eclair-core/src/main/scala/fr/acinq/eclair/api/NewService.scala

```diff
@@ -0,0 +1,251 @@
+                      complete(eclairApi.open(nodeId, fundingSatoshis, pushMsat, fundingFeerateSatByte, channelFlags))
+                  }
+                } ~
+                path("close") {
+                  formFields(channelIdNamedParameter, "scriptPubKey".as[ByteVector](binaryDataUnmarshaller).?) { (channelId, scriptPubKey_opt) =>
+                    complete(eclairApi.close(Left(channelId), scriptPubKey_opt))
+                  } ~ formFields(shortChannelIdNamedParameter, "scriptPubKey".as[ByteVector](binaryDataUnmarshaller).?) { (shortChannelId, scriptPubKey_opt) =>
+                    complete(eclairApi.close(Right(shortChannelId), scriptPubKey_opt))
+                  }
+                } ~
+                path("forceclose") {
+                  formFields(channelIdNamedParameter) { channelId =>
+                    complete(eclairApi.forceClose(Left(channelId)))
+                  } ~ formFields(shortChannelIdNamedParameter) { shortChannelId =>
+                    complete(eclairApi.forceClose(Right(shortChannelId)))
+                  }
+                } ~
+                path("updaterelayfee") {
+                  formFields(channelIdNamedParameter, "feeBaseMsat".as[Long], "feeProportionalMillionths".as[Long]) { (channelId, feeBase, feeProportional) =>
+                    complete(eclairApi.updateRelayFee(channelId.toString, feeBase, feeProportional))
+                  }
+                } ~
+                path("peers") {
+                  complete(eclairApi.peersInfo())
+                } ~
+                path("channels") {
+                  formFields("toRemoteNodeId".as[PublicKey].?) { toRemoteNodeId_opt =>
+                    complete(eclairApi.channelsInfo(toRemoteNodeId_opt))
+                  }
+                } ~
+                path("channel") {
+                  formFields(channelIdNamedParameter) { channelId =>
+                    complete(eclairApi.channelInfo(channelId))
+                  }
+                } ~
+                path("allnodes") {
+                  complete(eclairApi.allnodes())
+                } ~
+                path("allchannels") {
+                  complete(eclairApi.allchannels())
+                } ~
+                path("allupdates") {
+                  formFields("nodeId".as[PublicKey].?) { nodeId_opt =>
+                    complete(eclairApi.allupdates(nodeId_opt))
+                  }
+                } ~
+                path("receive") {
+                  formFields("description".as[String], "amountMsat".as[Long].?, "expireIn".as[Long].?) { (desc, amountMsat, expire) =>
+                    complete(eclairApi.receive(desc, amountMsat, expire))
+                  }
+                } ~
+                path("parseinvoice") {
+                  formFields("invoice".as[PaymentRequest]) { invoice =>
+                    complete(invoice)
+                  }
+                } ~
+                path("findRouteByInvoice") {
```

Comment:
Consistency issue: some endpoints are all lowercase (`updaterelayfee`, `forceclose`, ...), some are camelCased (`findRouteByNode`, `sendToInvoice`).

I think we should go for camelCase everywhere.

Reply:
Addressed in 26cc39e